vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -135,7 +135,7 @@
       chdir: "{{ BMOPATH }}"
 
   - name: Reinstate Baremetal Operator Manager
-    shell: "mv {{ BMOPATH }}/config/manager/manager.yaml.orig {{ BMOPATH }}/config/manager/manager.yaml"
+    shell: "mv {{ BMOPATH }}/config/default/kustomization.yaml.orig {{ BMOPATH }}/config/default/kustomization.yaml"
```

Comment:
Same question, why we are moving root kustomization.yaml instead of a specific file?